docs: Add `n_jobs` and `memory_map_size` to the status reference (#1700)
groonga/groonga: doc/source/reference/commands/status.rst

             "version": "2.0.0"
          }

   * - ``memory_map_size``

>      - The total memory map size in bytes of Groonga.

Reviewer: ```suggestion
     - The total mapped memory size by this Groonga process in bytes.
```
